packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -39,20 +40,26 @@ Sentry.init({
   integrations: [new BrowserTracing()],
   tracesSampleRate: 0.025,
   maxValueLength: 0,
-  beforeSend: event => {
+  beforeSend: (event, hint) => {
     if (event.message) {
       if (
         // Ignore events related to failed `eth_gasPrice` calls
         event.message.match(/eth_gasPrice/i) ||
         // Ignore events related to failed `eth_getBalance` calls
         event.message.match(/eth_getBalance/i) ||
         // Ignore events related to failed walletConnect calls
-        event.message.match(/Attempt to connect to relay via/i)
+        event.message.match(/Attempt to connect to relay via/i) ||
+        // Ignore events about window.propertyX being redefined accross multiple extensions
+        event.message.match(/Cannot redefine property/i)
```

Comment:
Just to be safe, should we also add a match for user-rejected-error here? Are we sure it can be detected only as a part of `hint`?

Reply:
Or, regarding the next line, shouldn't this be `isUserRejectedError(event.message)`?

Reply:
`isUserRejectedError` requires an error object, we check for the code, if it's an instance of `UserRejectedRequestError`, and then the message

https://github.com/OffchainLabs/arbitrum-token-bridge/blob/sentry-cleanup/packages/arb-token-bridge-ui/src/util/isUserRejectedError.ts#L17-L18